js/src/forum/index.js

```diff
@@ -17,27 +17,25 @@ import MarkdownButton from './components/MarkdownButton';
 app.initializers.add('flarum-markdown', function(app) {
   let index = 1;
 
-  extend(TextEditor.prototype, 'init', function() {
+  extend(TextEditor.prototype, 'oninit', function() {
     this.textareaId = 'textarea'+(index++);
   });
 
   extend(TextEditor.prototype, 'view', function(vdom) {
     vdom.children[0].attrs.id = this.textareaId;
   });
 
-  extend(TextEditor.prototype, 'configTextarea', function(value, element, isInitialized, context) {
-    if (isInitialized) return;
-
-    const editor = new MarkdownArea(element, {
+  extend(TextEditor.prototype, 'textareaOncreate', function(value, vnode) {
```

Comment:
I am confused. This method (and the `Onremove` equivalent) don't seem to exist in core, in the `mithril-2-update` branch? :thinking:

Reply:
This was changed in https://github.com/flarum/core/pull/2255/commits/bedf019bdb96aefd2c3827feaaae114a85b90319

An update is necessary here

Reply:
Fixed, tested locally.